diff -urN libiconv-1.8.orig/src/Makefile.in libiconv-1.8/src/Makefile.in
--- libiconv-1.8.orig/src/Makefile.in	2002-05-29 16:04:10.000000000 +0200
+++ libiconv-1.8/src/Makefile.in	2003-09-10 17:53:18.000000000 +0200
@@ -77,8 +77,8 @@
 	if [ ! -d $(DESTDIR)$(exec_prefix) ] ; then mkdir $(DESTDIR)$(exec_prefix) ; fi
 	if [ ! -d $(DESTDIR)$(bindir) ] ; then mkdir $(DESTDIR)$(bindir) ; fi
 	case "@host_os@" in \
-	  hpux*) $(CC) $(LDFLAGS) iconv.@OBJEXT@ -L$(libdir) -liconv @LIBINTL@ -o iconv;; \
-	  *) $(LIBTOOL_LINK) $(CC) $(LDFLAGS) iconv.@OBJEXT@ $(libdir)/libiconv.la @LTLIBINTL@ -o iconv;; \
+	  hpux*) $(CC) $(LDFLAGS) iconv.@OBJEXT@ -L$(DESTDIR)$(libdir) -liconv @LIBINTL@ -o iconv;; \
+	  *) $(LIBTOOL_LINK) $(CC) $(LDFLAGS) iconv.@OBJEXT@ $(DESTDIR)$(libdir)/libiconv.la @LTLIBINTL@ -o iconv;; \
 	esac
 	$(LIBTOOL_INSTALL) $(INSTALL_PROGRAM) iconv $(DESTDIR)$(bindir)/iconv
 
